prioritization_fees/src/account_prio_service.rs

```diff
@@ -0,0 +1,61 @@
+use solana_lite_rpc_core::types::BlockStream;
+use solana_sdk::pubkey::Pubkey;
+use tokio::task::JoinHandle;
+
+use crate::{account_priofees::AccountPrioStore, rpc_data::AccountPrioFeesStats};
+use tokio::sync::broadcast::error::RecvError::{Closed, Lagged};
+
+#[derive(Clone)]
```

Comment:
cloning service + store feels bit wrong but maybe that's just my preference

Reply:
Should be ok, the store is not cloned just a pointer.